app/views/organizations/dashboard/index.html.erb

```diff
@@ -0,0 +1,104 @@
+ <!-- Sidebar -->
+ <nav class="navbar-vertical navbar">
+     <div class="vh-100" data-simplebar>
+         <!-- Brand logo -->
+         <a class="navbar-brand" href="@@webRoot/index.html">
+             <img src="@@webRoot/assets/images/brand/logo/logo-inverse.svg" alt="" >
+         </a>
+         <!-- Navbar nav -->
+         <ul class="navbar-nav flex-column" id="sideNavbar">
+          <% if current_user.staff_account && current_user.staff_account.has_role?(:admin, current_user.staff_account.organization) %>
+            <li class="nav-item">
+                    <a class="nav-link @@if (context.page_group !== 'courses') { collapsed }" href="#"
+                        data-bs-toggle="collapse" data-bs-target="#navCourses" aria-expanded="false"
+                        aria-controls="navCourses">
+                        <i class="nav-icon fe fe-book me-2"></i> Organization
+                    </a>
+                    <div id="navCourses" class="collapse @@if (context.page_group === 'courses') { show }"
+                        data-bs-parent="#sideNavbar">
+                        <ul class="nav flex-column">
+                            <li class="nav-item">
+                                <a class="nav-link @@if (context.page === 'allcourses') { active }"
+                                    href="@@webRoot/pages/dashboard/admin-course-overview.html">
+                                    Settings
+                                </a>
+                            </li>
```

Comment:
When we get to work on this page let's make sure to use https://github.com/comfy/active_link_to which makes things a bit easier to show active states on links.

Reply:
Yes the hrefs in there right now are dropped in from the template. All need replacing.